packages/config/config/devices/0x0086/templates/aeotec_template.json

```diff
@@ -1127,5 +1323,529 @@
 		"valueSize": 3,
 		"unit": "%",
 		"defaultValue": 50
+	},
+	"wake_up_timeout": {
+		"label": "Wake Up Timeout",
+		"unit": "seconds",
+		"valueSize": 1,
+		"minValue": 8,
+		"maxValue": 127,
+		"defaultValue": 15,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": true
+	},
+	"current_power_mode": {
+		"$if": "firmwareVersion >= 1.7 && firmwareVersion <= 1.9",
```

Comment:
This seems to be specific to one device. Let's at least mention which one, either as a comment or in the template name.